src/VerifyCustomFactorController.js

```diff
@@ -57,6 +67,9 @@ export default FormController.extend({
             }
           })
           .catch(function(err) {
+            if (self.settings.get('features.skipIdpFactorVerificationBtn')) {
+              self.trigger('error', self, err);
```

Comment:
same question here, why do we need the extra trigger? If we throw an error, the form should show the error already. Is that not true?

Reply:
@mauriciocastillosilva-okta those (request and trigger) are modelEvents I added to start and stop the loading spinner.

When request is triggered `_startVerification ` is called and when `error` is triggered `_stopVerification `. This is a pattern employed in other controllers of the signin-widget as well.

Reply:
yeah, I see you added them, but I don't think we need them. Take a look at [BaseLoginModel](https://github.com/okta/okta-signin-widget/blob/master/src/models/BaseLoginModel.js#L59), in `manageTransaction` there is already a catch for any error and we trigger `error` there, so this will be a duplicate event we are triggering here